scala-package/scalastyle-config.xml

```diff
@@ -64,7 +64,7 @@ You can also disable only one rule, by specifying its rule id, as specified in:
 
   <check level="error" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
     <parameters>
-      <parameter name="maxLineLength"><![CDATA[100]]></parameter>
+      <parameter name="maxLineLength"><![CDATA[132]]></parameter>
```

Comment:
please raise a discussion on dev@ for changes that impact the entire project. 
There was a previous discussion on this topic and was agreed to keep at 100.
https://lists.apache.org/thread.html/3111d94f12f29ec834ae1b3184cf1ba2caafa1264b6b11736c4f02a4@%3Cdev.mxnet.apache.org%3E

Reply:
I'll revert this and take up the fight on there (missed that there was a discussion on this).